src/persistence/job.js

```diff
@@ -0,0 +1,32 @@
+const sql = require('sql-template-strings');
+const {v4: uuidv4} = require('uuid');
+const db = require('./db');
+
+module.exports = {
+  // Method create() will get all information about the job without id and create a new job with those data
```

Comment:
The same comment above. The rule is comment restriction.

Reply:
Updated.